synthtool/gcp/templates/python_library/noxfile.py.j2

```diff
@@ -139,6 +138,7 @@ nox.options.sessions = [
     "lint_setup_py",
     "blacken",
     "docs",
+    "docfx",
```

Comment:
Question: What changed that it wasn't needed before but is now?

Reply:
Some repositories had a separate docfx Kokoro job but we're slowly removing that presubmit in favor of running `docfx` via the `Kokoro` job or via `Github Actions`